### PR TITLE
177 admin login country state district address fields order change

### PR DESCRIPTION
**Pull request by anand-it**
admin login country state district in DropDown
image
image

### CODE REVIEW

File: src/app/admin/modules/staff/add-staff/add-staff.component.ts [hunk 1]
```diff
@@ -143,4 +148,32 @@ export class AddStaffComponent {
...
+          this.stateList = data.data.states
+        }
+      },
+      (error: any) => {
+        this.apiService.showNotification('snackbar-danger', 'Error!!!');
+      }
+    );
+  }
+
+  getStateByCity() {
```
Comment (kumaresan-subramani) [inline src/app/admin/modules/staff/add-staff/add-staff.component.ts:164]: getCityByState - pls do rename

File: src/app/admin/modules/staff/add-staff/add-staff.component.ts [hunk 2]
```diff
@@ -143,4 +148,32 @@ export class AddStaffComponent {
   updateProofUrl(newItem: any) {
     this.staffForm.value.proof_image = newItem;
   }
+
+  getCountryByState() {
```
Comment (kumaresan-subramani) [inline src/app/admin/modules/staff/add-staff/add-staff.component.ts:152]: getStateByCountry - pls rename

**Review by kumaresan-subramani — APPROVED** [review]

File: src/app/admin/modules/staff/add-staff/add-staff.component.ts (hunk 2 above)
Comment (anand-it) [reply src/app/admin/modules/staff/add-staff/add-staff.component.ts]: Done

File: src/app/admin/modules/staff/add-staff/add-staff.component.ts (hunk 1 above)
Comment (anand-it) [reply src/app/admin/modules/staff/add-staff/add-staff.component.ts]: Done

**Pull request merged**